packages/grid/test/keyboard-navigation-row-focus.common.js

```diff
@@ -101,7 +101,7 @@ function getTabbableRows(root) {
 
 function hierarchicalDataProvider({ parentItem }, callback) {
   // Let's use a count lower than pageSize so we can ignore page + pageSize for now
-  const itemsOnEachLevel = 5;
+  const itemsOnEachLevel = 100;
```

Comment:
nit: I guess this is no longer lower than page size so the comment no longer applies?

Reply:
Oh, good catch. I forgot to add pagination. Fixed.